package.json

```diff
@@ -69,7 +69,7 @@
     "graphql-middleware": "4.0.3",
     "graphql-parse-resolve-info": "^4.12.3",
     "graphql-relay": "0.7.0",
-    "graphql-tools": "4.0.8",
+    "graphql-tools": "5",
```

Comment:
I started to see the light at the end of the tunnel here. I figured out why not make this upgrade here to avoid all the previous enums drama since the community mentions that they fixed that in v5 and see what's breaking.
- I figured out there were some API changes around stitching, which I've made in this commit that you can scroll until end of the file to see them.
- Obviously, I had to removed graphql-tools patch next and updated the enums to be defined as described in the docs.

And that's it 🎉 
The next commits fix issues that are present in this commit:
[Issue with FeatureFlagSortEnums] https://github.com/artsy/metaphysics/pull/5838/commits/2a7c2978c5030a19c042ce264f4cc07c3ecc2f57
[Forgot to transform Date from exchange] https://github.com/artsy/metaphysics/pull/5838/commits/da9cc71b160ba6218c81fb0f6653a9ae5a4d7bf3
[**Fix broken stitching changes**] https://github.com/artsy/metaphysics/pull/5838/commits/38f51fdd10c12492d4000473e49317f7cb499a38 this one requires some attention - [**Go to issue**]